spec/unit/virtus/attribute/collection/value_coerced_predicate_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+require 'set'
+
+describe Virtus::Attribute::Collection, '#value_coerced?' do
+  subject { object.value_coerced?(input) }
+
+  let(:object) { described_class.build(Array[Integer]) }
+
+  context 'when input has correctly typed members' do
+    let(:input) { [1,2,3] }
```

Comment:
I don't personally use Rubocop but you don't need Hound CI, this repo uses a public CI somewhere and all CI's can be adjusted to run a custom command that does both and fails a test. That said I don't know if we need Rubocop.